examples/plotting/data_models.py

```diff
@@ -1,3 +1,10 @@
+'''This example shows creating sinusoidal wave using bokeh DataModel.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.model.DataModel, bokeh.models.ColumnDataSource, bokeh.models.CustomJS, bokeh.core.properties.Float
```

Comment:
typo and suggestion:
```suggestion
    :apis: bokeh.models.DataModel, bokeh.models.ColumnDataSource, bokeh.models.CustomJS
```